be/src/common/config.cpp

```diff
@@ -1194,6 +1194,7 @@ DEFINE_mInt32(table_sink_partition_write_max_partition_nums_per_writer, "128");
 
 /** Hive sink configurations **/
 DEFINE_mInt64(hive_sink_max_file_size, "1073741824"); // 1GB
+DEFINE_mInt32(hive_sink_max_open_partitions, "10000");
```

Comment:
I think 10000 is too many? How about 2000?

Reply:
will use `table_sink_partition_write_max_partition_nums_per_writer`